<?xml version="1.0" encoding="utf-8"?>
<launch>
  <param name ="/use_sim_time" value="true"/>

  <param name = "result_file" type="string" value="$(find cam_lidar_calib)/result/C_T_L_basler.txt"/>

  <param name= "project_only_plane" type="bool" value="false"/>

  <param name = "camera_in_topic" type="string" value="/pylon_camera_node/image_raw"/>
  <param name = "lidar_in_topic" type="string" value="/velodyne_points"/>
  <param name = "camera_name" type="string" value="basler"/>
  
  <param name = "dist_cut_off" type="int" value="5"/>

  <param name = "cam_config_file_path" type="string" value="$(find cam_lidar_calib)/config/basler_config.yaml" />

  <node  pkg="cam_lidar_calib" name="cam_lidar_proj_basler" type="cam_lidar_proj_node" output="screen">
  </node>
</launch>
